xinference/model/llm/llama_cpp/core.py

```diff
@@ -302,7 +302,8 @@ def _to_iterator():
                 while (r := q.get()) is not _Done:
                     if type(r) is _Error:
                         raise Exception("Got error in chat stream: %s", r.msg)
-                    yield r
+                    chunk_keys = ChatCompletionChunk.__annotations__.keys()
```

Comment:
Could you add some comment?

e.g. the chunk contains some key like `system_fingerprint` this may cause xxx.

thus we need to filter it.

Reply:
And you don't need to get keys()

```
key in ChatCompletionChunk.__annotations__
```

is more clean and efficient.